gamestonk_terminal/options/fdscanner_view.py

```diff
@@ -0,0 +1,43 @@
+"""Fdscanner view"""
+__docformat__ = "numpy"
+
+import os
+
+from tabulate import tabulate
+import pandas as pd
+
+from gamestonk_terminal.options import fdscanner_model
+from gamestonk_terminal.helper_funcs import export_data
+
+
+def display_options(num: int, sort_column: pd.Timestamp, export: str, ascending: bool):
+    """Displays the unusual options table
+
+    Parameters
+    ----------
+    num: int
+        Number of rows to show
+    sort_columns: pd.Timestamp
+        Data column to sort on
+    export: str
+        File type to export
+    ascending: bool
+        Whether to sort in ascending order
+    """
+    data, last_update = fdscanner_model.unusual_options(num, sort_column, ascending)
+
+    if export:
+        export_data(
+            export,
+            os.path.dirname(os.path.abspath(__file__)),
+            f"UnusualOptions_updated_{last_update}",
```

Comment:
Here should be the name of the command. 

The export_data func already appends a timestamp to the file

Reply:
The new time stamp actually tells you when the options were updated.  You could run the command 20 minutes apart and get the same data

Reply:
I think that it would be better to keep this as the name of the command, since this is what we're going to do with all commands. Also having 2 timestamps is a bit ugly.

<img width="442" alt="Captura de ecrã 2021-08-03, às 21 31 41" src="https://user-images.githubusercontent.com/25267873/128082066-fdd402b8-a018-48c7-b29b-0379715531b4.png">

Most of the commands rely on a certain update time from the server side. So, the user knows about this when they export.